openai_files/helpers.py

```diff
@@ -10,8 +10,18 @@ def get_system_message(prompt_type: PromptType) -> str:
 
 
 PROMPTS = {
+    PromptType.GENERATE_IMAGE_HAPPY: lambda holiday_list: (
+        f"Here is a list of holidays: {holiday_list}\n"
+        "Please create an image that combines all these events with the context of a happy crypto "
+        "investor as the subject."
+    ),
+    PromptType.GENERATE_IMAGE_SAD: lambda holiday_list: (
+        f"Here is a list of holidays: {holiday_list}\n"
+        "Please create an image that combines all these events with the context of a sad crypto "
+        "investor as the subject, but make it a little bit funny."
```

Comment:
ah yeah, i thought about maybe adding a `mood` parameter but just decided to go with 2 separate ones for a couple of reasons. first, i thought we might want to modify one mood prompt and not the other and this would suppor that. Second, if we wanted to add another parameter to this prompt, it would make things a bit messier with the way the prompt generator was designed. For example, if we added another parameter:

`PromptType.GENERATE_IMAGE: lambda holiday_list, mood: f"Here is a list of holidays {holiday_list} and make it {mood}."`

it would make the `GET_HOLIDAYS` prompt messier since we have to make sure all functions within this dict take the max number of parameters:
`PromptType.GET_HOLIDAYS: lambda date, _: ...`

and we'd have to modify `get_prompt()`:

```python
def get_prompt(prompt_type: PromptType, param: str, param2: str) -> str:
    return PROMPTS[prompt_type](param, param2)
   ```
or use `**kwargs` or something

```python
def get_prompt(prompt_type: PromptType, **kwargs: dict[str, Any]) -> str:
    return PROMPTS[prompt_type](**kwargs)
```

But then it would still make the prompt a bit messier since we'd have to `kwargs.get()` the values.

Maybe we might find a reason to do that in the future, but for readability and to be able to edit sad/happy prompts separately, I went with a single param.